docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -0,0 +1,68 @@
+# Deploying ASO v2 in multi-tenant mode
+
+The current release version of the ASO deployment YAML installs the operator in single-tenant mode:
+it's deployed in the `azureserviceoperator-system` namespace with one set of Azure credentials, and manages resources in any namespace in the cluster.
+That single operator deployment handles webhooks fired when Azure resources are changed.
+
+ASO can also be deployed in a multi-tenant configuration that enables using separate credentials to manage resources in different Kubernetes namespaces.
+
+Running the operator in multi-tenant mode requires one deployment to handle webhooks (since webhook configurations are cluster-level resources) and then a deployment for each tenant, each with its own credentials and set of namespaces that it watches for Azure resources.
+To deploy the operator in multi-tenant mode the release YAML has been split into two parts:
+
+1. Cluster-wide resources:
+   * Custom resource definitions for the Azure resources
+   * Cluster roles for managing those resources
+   * The `azureserviceoperator-system` namespace containing the deployment and service to run the operator.
+     The webhook deployment of the operator is configured to run in webhook-only mode.
```

Comment:
```suggestion
     The webhook deployment if the operator is configured to run in webhook-only mode.
```

I think?

Reply:
Combine with the above comment though I think it might be clearer to just talk about this as the webhooks for the multi-tenant operator (or something) though, rather than calling it "the operator" which confused me a bit because there's kinda more than one of them.

Reply:
Yeah, I agree - I'll reword it to make that clearer.